_episodes/02-git.md

```diff
@@ -334,16 +383,25 @@ $ git commit -m "Initial commit after CMS Cookiecutter creation, version 1.0"
 
 ## Creating new features - using branches
 
-When you are working on a project to implement new features, it is a good practices to isolate the the changes you are making and work on one particular topic at a time. To do this, you can use something called a **branch** in git. Working on branches allows you to isolate particular changes. If you make sure that your code works before merging to your main or **master** branch, you will ensure that you always have a working version of code on your main branch.
+When you are working on a project to implement new features,
+it is a good practice to isolate the changes you are making and work on one particular topic at a time.
+To do this, you can use something called a **branch** in git.
+Working on branches allows you to isolate particular changes.
+If you make sure that your code works before merging to your main or **master** branch,
```

Comment:
@sjayellis @eirrgang @radifar - This PR is currently correct in that "master" will be the branch name. However, one thing I meant to bring up in set-up is adding switching the default branch name to "main" in the initial git config. It can be achieved with this line :

```
git config --global init.defaultBranch main
```

Are there any objections to making this change? If not, Sam, could you add it to the set-up with the rest of the git config settings?

Reply:
I think that the addition of this set-up detail should occur in the same commit / PR as an update to the references to `master` elsewhere in the material. Since Sam is taking charge of the git/setup updates, I will leave this unaltered. Merge conflicts will be easy enough to resolve.